src/applications/vaos/utils/constants.js

```diff
@@ -12,6 +12,19 @@ export const APPOINTMENT_TYPES = {
   ccRequest: 'ccRequest',
 };
 
+export const APPOINTMENT_STATUS = {
+  pending: 'pending',
+  booked: 'booked',
+  cancelled: 'cancelled',
```

Comment:
This is totally a nit and I know that the API returns "Cancelled" with 2 L's, but should we try to use "canceled" with 1 L throughout for consistency, since that is the more common spelling in American English?

Reply:
I ended up choosing the two L version because the list from http://hl7.org/fhir/dstu2/valueset-appointmentstatus.html uses two Ls. So I thought using that everywhere except for the text visible to the user would be easier for us. Though either way seems workable.